src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

```diff
@@ -321,7 +321,7 @@ export function useSearchCurrencyLists() {
   const { data: verifiedAssets, isLoading: isLoadingVerifiedAssets } = useTokenSearch(
     {
       list: 'verifiedAssets',
-      chainId: isAddress(query) ? state.toChainId : undefined,
+      chainId: state.toChainId,
```

Comment:
without this change, the v3 upgrade was showing consolidated repeated results (same token across multiple networks)